Article/Article.js

```diff
@@ -112,3 +156,60 @@ const data = [
   Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new artible
 
 */
+
+window.addEventListener("load", e => {
+  const articles = document.querySelector(".articles");
+
+  data.forEach(element => {
+    articles.appendChild(
+      createArticle(
+        element.title,
+        element.date,
+        element.firstParagraph,
+        element.secondParagraph,
+        element.thirdParagraph
+      )
+    );
+  });
+
```

Comment:
You have the e for event parameter but you are not using. Since you don't need it you can remove it.